os/include/tinyara/fs/mtd.h

```diff
@@ -237,6 +239,16 @@ FAR struct mtd_dev_s *mtd_partition(FAR struct mtd_dev_s *mtd, off_t firstblock,
  ****************************************************************************/
 FAR struct mtd_dev_s *get_mtd_partition(uint16_t tagno);
```

Comment:
Currently, we can get only `MTD_FS`, `MTD_ROMFS` and `MTD_FTL` type.
Please add description about this.

Reply:
I think I don't need to add description about that because it is not that this function doesn't support other values.